src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -858,6 +865,14 @@ private void setWinningRulesDefaultValues() {
   }
 
   private void setDefaultValues() {
+    String versionText = "Running %s version %s".formatted(Main.APP_NAME, Main.APP_VERSION);
+    if (Main.APP_VERSION.endsWith("999")) {
+      versionText += " -- this is a development version";
+      labelVersion.setBackground(new Background(new BackgroundFill(
+          Color.DARKRED, CornerRadii.EMPTY, Insets.EMPTY)));
+      labelVersion.setTextFill(Color.WHITE);
+    }
+    labelVersion.setText(versionText);
```

Comment:
There are some truncation issues with both `labelVersion` and `labelCurrentlyLoaded` when you load a config, but maybe it's ok?

Reply:
Yeah, I figure since the full text is there on launch, it becomes less important after you load a config and some truncation is okay then? It was the easiest way I could come up with to not take up more screen real estate.